activerecord/test/cases/adapter_test.rb

```diff
@@ -14,6 +14,19 @@ def setup
       @connection.materialize_transactions
     end
 
+    def test_type_map_is_ractor_shareable
+      # This is testing internals. Please feel free to remove this test
+      # or change it when internals change. The point is to make sure
+      # the type map is Ractor shareable.
+      skip unless defined?(Ractor)
```

Comment:
Is this even needed?

Reply:
I assume it's needed for implementations besides CRuby? I'm not sure though. I'll try removing the line and see if CI breaks.

Reply:
Hum, we don't test nor claim to support any other implementation. But yes, it's true JRuby doesn't have Ractors AFAIK.